TransferMethod/Tests/SelectTransferMethodTypePresenterTests.swift

```diff
@@ -197,50 +197,56 @@ class SelectTransferMethodTypePresenterTests: XCTestCase {
 
     func testNavigateToAddTransferMethod_success() {
         // Given
-        addGetHyperwalletUserResponse(fileName: "UserBusinessResponse")
+        addGetIndividualHyperwalletUserResponse()
         HyperwalletTestHelper.setUpMockServer(request: setUpTransferMethodConfigurationKeys())
 
-        let countryIndex = 0
-        let currencyIndex = 1
-        let sectionDataCount = 3
-        let firstIndexPath = IndexPath(row: 0, section: 0)
-        let secondIndexPath = IndexPath(row: 1, section: 0)
         let expectation = self.expectation(description: "load transfer methods keys")
         mockView.expectation = expectation
 
         presenter.loadTransferMethodKeys(true)
         wait(for: [expectation], timeout: 1)
 
-        mockView.ignoreXCTestExpectation = true
-        presenter.performShowSelectCountryOrCurrencyView(index: countryIndex)
-
-        presenter.performShowSelectCountryOrCurrencyView(index: currencyIndex)
-
         // When
         presenter.navigateToAddTransferMethod(0)
 
         // Then
         XCTAssertEqual(presenter.selectedCountry, "US", "The country should be US")
-        XCTAssertEqual(presenter.selectedCurrency, "USD", "The currency should be USD")
+        XCTAssertEqual(presenter.selectedCurrency, "CAD", "The currency should be CAD")
         XCTAssertTrue(mockView.isNavigateToAddTransferMethodControllerPerformed,
                       "The navigateToAddTransferMethodControllerPerformed should be performed")
 
-        XCTAssertEqual(mockView.profileType!, "BUSINESS", "The profileType should be BUSINESS")
+        XCTAssertEqual(mockView.profileType, "INDIVIDUAL", "The profileType should be INDIVIDUAL")
         XCTAssertEqual(presenter.countryCurrencySectionData.count, 2, "The countryCurrencyCount should be 2")
-        XCTAssertEqual(presenter.sectionData.count, sectionDataCount, "The transferMethodTypesCount should be 3")
-        XCTAssertNotNil(presenter
-            .sectionData[firstIndexPath
-                .row] as HyperwalletTransferMethodType, "The cell configuration should not be nil")
-        XCTAssertNotNil(presenter
-            .sectionData[secondIndexPath
-                .row] as HyperwalletTransferMethodType, "The cell configuration should not be nil")
-
-        XCTAssertNotNil(presenter.getCountryCurrencyConfiguration(indexPath: firstIndexPath),
-                        "The country currency cell configuration should not be nil")
-        XCTAssertNotNil(presenter.getCountryCurrencyConfiguration(indexPath: secondIndexPath),
-                        "Out of bounds country currency configuration should not be nil")
+        XCTAssertEqual(presenter.sectionData.count, 1, "The transferMethodTypesCount should be 1")
+        XCTAssertTrue(mockHyperwalletInsights.trackImpression, "HyperwalletInsights.trackImpression should be called")
```

Comment:
It should be trackClick instead of trackImpression

Reply:
trackClick added